### DISCUSSION TITLE
@Montorni you have to create it the file.

### DISCUSSION THREAD

**Discussion by MadokaMagicas** [Monika-After-Story/MonikaModDev#9140] [QA - tech support + general] [answered]
@Montorni you have to create it the file.

_Originally posted by @ThePotatoGuy in https://github.com/Monika-After-Story/MonikaModDev/issues/1275#issuecomment-405454453_

HOW DO I CREATE IT SORRY I’M DUMB 😭😭

**Comment by ThePotatoGuy** [comment, marked as answer]
1. create the text file
2. open text file in notepad
3. paste the text as desired.
